### PR TITLE
adding tab for c/c++

### PR DESCRIPTION
**Pull request by PoojanSmart**
adding tab for c/c++

### CODE REVIEW

File: content/motivation.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ to establish accuracy:
        expected_result = 37.777777
        assert abs(temp_c - expected_result) < 1.0e-6
    ```
+   ```{code-tab} c/c++
```
Comment (robertodr) [inline content/motivation.md:49]: You need `c++` as argument, otherwise the Pygments lexer gets confused on how to generate highlighting

Comment (PoojanSmart) [reply content/motivation.md]: Hey, I have updated the code and added cpp code also. please review and merge

**Review by robertodr — APPROVED** [review]
Thanks!

**Comment by robertodr** [comment]
@wikfeldt I do not have permissions to merge

**Comment by wikfeldt** [comment]
thanks @PoojanSmart for the contribution and @robertodr for the review!

**Pull request merged**